features/filiatie.feature

```diff
@@ -0,0 +1,470 @@
+            En bevat het antwoord geen property "bijbehorendeAppartementsrechtIdentificaties"
+    
+    Rule: Na vereniging van percelen wordt filiatie getoond bij de vervallen percelen en bij het nieuwe perceel
+        Scenario: Opvragen vervallen perceel na vereniging
+            Als "/kadastraalonroerendezaken/P4" wordt gevraagd
+            Dan bevat het antwoord "isOvergegaanIn" met waarde:    
+                """
+                [
+                    {
+                        "aard": {
+                            "code": "19",
+                            "waarde": "Verenigen percelen"
+                        },
+                        "identificatie": "P6"
+                    }
+                ]
+                """
+            En bevat het antwoord "indicatieVervallen": true
+            En bevat het antwoord property "einddatum" met een waarde
+            En bevat het antwoord geen property "indicatieSluimerend"
+            En bevat het antwoord geen property "bijbehorendeGrondperceelIdentificaties"
+            En bevat het antwoord geen property "bijbehorendeAppartementsrechtIdentificaties"
+
+        Scenario: Opvragen perceel uit vereniging
+            Als "/kadastraalonroerendezaken/P6" wordt gevraagd
+            Dan bevat het antwoord "isOntstaanUit" met waarde:    
+                """
+                [
+                    {
+                        "aard": {
+                            "code": "19",
+                            "waarde": "Verenigen percelen"
+                        },
+                        "identificatie": "P4"
+                    },
+                    {
+                        "aard": {
+                            "code": "19",
+                            "waarde": "Verenigen percelen"
+                        },
+                        "identificatie": "P5"
+                    }
+                ]
+                """
+            En bevat het antwoord geen property "indicatieVervallen"
+            En bevat het antwoord geen property "einddatum"
+            En bevat het antwoord geen property "indicatieSluimerend"
+            En bevat het antwoord geen property "isOvergegaanIn"
+            En bevat het antwoord geen property "bijbehorendeGrondperceelIdentificaties"
+            En bevat het antwoord geen property "bijbehorendeAppartementsrechtIdentificaties"
+
+    Rule: Splitsing in appartementsrechten van een perceel toont verwijzingen van het grondperceel naar de appartementsrechten en vice versa
+        Scenario: Opvragen van het grondperceel na splitsing in appartementsrechten
+            Als "/kadastraalonroerendezaken/P7" wordt gevraagd
+            Dan bevat het antwoord "isOvergegaanIn" met waarde:    
+                """
+                [
+                    {
+                        "identificatie": "A1"
+                    },
+                    {
+                        "identificatie": "A2"
+                    }
+                ]
+                """
```

Comment:
Moet hier dan ook niethet appartementsrechtVolgnummer getoond worden ?
Die geldt overal voor isOvergegaanIn en isOntstaanUit.
Of hoort dat bij de wijziging a.g.v. de structuur die nog openstaat ?

Als het appartemenstrechtVolgnummer niet getoond hoeft te worden dan staat het misschien onterecht in de OAS3 (misschien dat de discussies elkaar gekruist hebben.)

Reply:
volgnummer nemen we nu alleen op in de bijbehorendeAppartementsrechten toch?